src/modules/Lan.rb

```diff
@@ -866,14 +866,14 @@ def ProposeVirtualized
 
         bridge_name = format("br%s", NetworkInterfaces.GetFreeDevice("br"))
 
-        if !LanItems.IsBridgeable(bridge_name, current)
-          log.info "The interface #{ifcfg} is not bridgeable."
+        if !LanItems.IsBridgeable(bridge_name, current) || !proposable_as_bridge?(config)
+          log.info "The interface #{ifcfg} can not be proposed as bridge."
           next
         end
 
         LanItems.current = current
 
-        propose_current_item!(config) if !LanItems.IsCurrentConfigured
+        LanItems.ProposeItem if !LanItems.IsCurrentConfigured
```

Comment:
I'd put ```!LanItems.IsCurrentConfigured``` into ```!proposable_as_bridge?(config)``` and name it somehow like ```propose_bridge?``` and result use in this condition

Reply:
The naming is better, and probably what has to be changed is the `IsBridgeable` check adding a convenience method which adds to the `IsBridgeable` checks the `carrier` check and the hwinfo `wlan` check

Reply:
these checks are out ```IsBridgeable``` bcs user can select a device without link explicitly if needed.